plugins/generator-1.20.1/forge-1.20.1/templates/livingentity/livingentity.java.ftl

```diff
@@ -55,6 +56,16 @@ import javax.annotation.Nullable;
 
 public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements RangedAttackMob</#if> {
 
+	<#list data.entityDataEntries as entry>
+		<#if entry.value().getClass().getSimpleName() == "Integer">
+			public static final EntityDataAccessor<Integer> ${entry.property().getName()} = SynchedEntityData.defineId(${name}Entity.class, EntityDataSerializers.INT);

```

Comment:
It may make sense to give some prefix to those entries, eg. vanilla adds DATA_ to most of those cases

Reply:
✔️ Fixed

Reply:
I believe this also works